src/nova-driver.md

```diff
@@ -0,0 +1,54 @@
+# Nova GPU Driver
+
+Nova is a driver for GSP-based Nvidia GPUs that is currently under development
+and is being written in Rust.
+
+As of today, the driver is in the (very) early stage and not yet operational.
```

Comment:
I would probably describe that getting a bunch of Rust abstractions upstream is a prerequisite that the team focuses on initially. Hence, Novas first milestone is to hit upstream as a stub driver just implementing basic device / driver abstractions for PCI / DRM.